src/hyp3_isce2/metadata/templates/insar_burst/readme.md.txt.j2

```diff
@@ -0,0 +1,306 @@
+
+The unwrapped interferogram files are tagged with _unw_phase, and the _unw_phase.tif file contains the data values. The PNG is 2048 pixels wide.
+
+----------------
+
+## 3. Coherence Map
+
+The spectra of the two images used for InSAR must overlap well enough to generate interferometric fringes. Areas without common overlap must be filtered out prior to InSAR generation. The correlation indicates the accuracy of the phase information, or the visibility of fringes. Areas with low correlation will have noisier phase. The magnitude of the correlation is commonly referred to as coherence. The coherence map is output as a GeoTIFF with values ranging from 0 to 1; the larger the number, the higher the coherence (and the better the correlation).
+
+The coherence map is tagged with _corr.tif
+
+----------------
+## 4. Connected Components TODO: Populate content
+
+The connected components file is tagged with _conncomp.tif
+
+----------------
+## 5. Parameter Documentation
+
+The text file included with the product lists the key processing parameters used, including the filenames of the reference and secondary bursts used, baseline length, number of looks, output resolution, and source DEM information.
+
+The text file is not tagged; it has a .txt extension on the base filename.
+
+----------------
+## 6. Look Vector Maps TODO: Edit as appropriate
+
+Two look vector maps are output with as GeoTIFFs. The lv_theta map indicates the SAR look vector elevation angle at each map pixel, ranging from -pi/2 (down) to pi/2 (up). The lv_phi map indicates the SAR look vector orientation angle at each map pixel, ranging from -pi/2 (south) to pi/2 (north), with east being 0. Angles are expressed in radians, referencing the look direction back towards the sensor, and are calculated from a horizontal elevation plane at each pixel.
+
+The lv_theta map is required for applying atmospheric corrections, and for calculating vertical displacement values relative to a new phase unwrapping reference point.
+
+----------------
+
+## 7. DEM used to process the data
+
+This is a copy of the Digital Elevation Model (DEM) layer used for processing. This layer is tagged with _dem.tif, and is a 16-bit signed integer GeoTIFF.
+
+The source DEM has a geoid correction applied before it is used for processing, so height values in this file will differ from the source DEM. The DEM is down-sampled to twice the pixel spacing of the desired InSAR output pixel spacing to smooth it before processing. As such, an 80-m InSAR product requires the DEM to be down-sampled to a pixel spacing of 160 m. It is provided in the same pixel spacing/alignment and projected to the same UTM Zone as the interferogram files.
```

Comment:
@forrestfwilliams Verify that this is the correct treatment of the DEM. Do we still downsample?

Reply:
I don't think this downsample happens. Removing.